styles/floating-images.css

```diff
@@ -103,7 +103,7 @@ main .section.image-left .default-content-wrapper .content .button-container {
     }
 }
 
-@media (max-width: 600px) {
+@media (max-width: 599px) {
```

Comment:
Why the change?

Reply:
I did additional testing and find out that changes for floating image happens when we change dimensions from 599->600

Reply:
Is it going to break something if we settle on 600px for everything?

Reply:
nope. I think it should work fine

Reply:
Let's bump this to 600px and that should be last change for this.